server/image_pull_test.go

```diff
@@ -132,11 +132,7 @@ var _ = t.Describe("ImagePull", func() {
 
 		It("should fail credential decode erros", func() {
 			// Given
-			gomock.InOrder(
-				imageServerMock.EXPECT().ResolveNames(
-					gomock.Any(), gomock.Any()).
-					Return([]string{"image"}, nil),
-			)
+			gomock.InOrder()
```

Comment:
Please remove this line if not needed.

Reply:
Fixed, thanks. (Kept the `// Given` comment based on `utils/utils_test.go` doing the same.)